examples/example.prompt.sh

```diff
@@ -0,0 +1,81 @@
+# -*- mode: shell-script; mode: flyspell-prog; ispell-local-dictionary: "american" -*-
+#
+# Example PS1 prompt.
+#
+# Example - rename this to .drush_prompt, and in your ~/.bash_profile add:
+#
+#    if [ -f ~/.drush_prompt ] ; then
+#        . ~/.drush_prompt
+#    fi
+#
+# Features:
+#
+# Displays Git repository and Drush alias status in your prompt.
+
+if [ "\$(type -t __git_ps1)" ] && [ "\$(type -t __drush_ps1)" ]; then
+
+  # Uncomment the following line to enable color hints in your Drush prompt.
+  # Modify __drush_ps1_colorize_alias() to customize your color theme.
+  #
+  # DRUSH_PS1_SHOWCOLORHINTS=true
```

Comment:
Dont think this needs to be commented out. I would expect this file to work if I deliberately include it from .bashrc.